specs/posix/sudoers.table

```diff
@@ -0,0 +1,7 @@
+table_name("sudoers")
+description("Rules for running commands as other users via sudo.")
+schema([
+    Column("header", TEXT, "Symbol for given rule"),
+    Column("rule_details", TEXT, "Rule definition")
```

Comment:
Is there any additional parsing of the `rule_details` that might be beneficial?  Check out [this blog post](https://www.garron.me/en/linux/visudo-command-sudoers-file-sudo-default-editor.html), it seems to me like we could maybe carve up the `rule_details` data a bit more into more discrete columns?